src/components/formik-auto-fill/FormikAutoFill.tsx

```diff
@@ -108,24 +133,31 @@ export class FormikAutoFill extends React.Component<Props, State> {
       disabled: this.props.disabled,
     };
 
+    const spinnerContent = (
+      <Spinner color={color.getColor('primary')}/>
+    );
+
     return (
-      <WrapperStyled>
-        <LabelStyled error={this.props.error}
-                     disabled={this.props.disabled}>
-          {label}
-        </LabelStyled>
-
-        <Autocomplete inputProps={inputProps}
-                      items={this.state.data}
-                      value={this.state.value}
-                      renderInput={(props: any) => this.renderInput(props, inputProps.error, inputProps.disabled)}
-                      renderItem={(item: any, isHighlighted: boolean) => this.renderItem(item, isHighlighted)}
-                      getItemValue={(val: any) => this.getItemValue(val)}
-                      onChange={(val: any) => this.onChange(val)}
-                      onSelect={(val: any, item: Item) => this.onSelect(val, item)}/>
-
-        {touched && errors && <div>{errors}</div>}
-      </WrapperStyled>
+      <FlexStyled>
+         {spinnerContent}
```

Comment:
Je kan hier gewoon ```<Spinner color={color.getColor('primary')}/>``` zetten.

Reply:
Wanneer er een spinner verschijnt zal de autofill naar rechts springen => spinner absoluut positioneren. (Dit is nu ook fout bij de loading button)

Reply:
Wss is het de bedoeling nog steeds om de loader achteraf in de input te zetten? Zet hier dan even een todo comment bij.